go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/openshift/cluster-monitoring-operator
 
-go 1.23.0
+go 1.24.0
 
-toolchain go1.24.0
```

Comment:
Interesting, `go mod tidy/vendor` seem to bump these automatically.

Reply:
So my local was 1.24 which is why this had no effect. Using `gvm` to match this.